pipeline-model-declarative-agent/README.md

```diff
@@ -0,0 +1 @@
+This plugin is deprecated, please use [Pipeline: Declarative](https://plugins.jenkins.io/pipeline-model-definition/) plugin instead.
```

Comment:
I was confused about how this would show up on plugins.jenkins.io, but it looks like there is a way to override the URL used per-plugin that would be used to point to this file, see https://github.com/jenkins-infra/update-center2/pull/367.